scss/widgets/material/treeView/_index.scss

```diff
@@ -95,6 +103,10 @@ $material-treeview-item-with-checkbox-offset: $material-treeview-checkbox-offset
 }
 
 .dx-treeview-item-without-checkbox {
+  &.dx-treeview-item-with-custom-expander-icon > .dx-treeview-item {
+    padding-left: $material-treeview-item-side-padding;
```

Comment:
Why don't we want add this padding in generic theme?

Reply:
lets ask Pavel tomorrow)